air/src/air/aux.rs

```diff
@@ -60,70 +62,66 @@ impl<E> AuxRandElements<E> {
 /// the Lagrange kernel random elements. Those claims are (linearly) combined into one using the
 /// openings combining randomness.
 #[derive(Clone, Debug)]
-pub struct GkrRandElements<E> {
-    lagrange: LagrangeKernelRandElements<E>,
-    openings_combining_randomness: Vec<E>,
+pub struct GkrData<E: FieldElement> {
+    pub lagrange_kernel_eval_point: LagrangeKernelRandElements<E>,
+    pub openings_combining_randomness: Vec<E>,
+    pub openings: Vec<E>,
+    pub oracles: Vec<LogUpGkrOracle<E::BaseField>>,
```

Comment:
I don't think it is necessary to store the oracles here

Reply:
I think it is needed for the s-column, which is done in another PR building on this one